cypress/integration/smokeTests/prodOnly/search.ts

```diff
@@ -1,64 +0,0 @@
-/**
- * Remove this file after 1.13 deployed and update package.json "test-staging-no-auth"
- * and "test-prod-no-auth" to restore the sharedTests version of this file.
```

Comment:
Do you need to update the package.json?

Reply:
Good catch, all I needed to do was read my own comment.